applications/portal/frontend/src/components/viewers/detailsViewer/DetailsViewer.tsx

```diff
@@ -0,0 +1,483 @@
+                                    <span>Uploaded by</span>
+                                </div>
+                                <div className={classes.textBlock} style={{ alignItems: 'flex-end' }}>
+                                    <span>06 Nov 2023, 11.58am</span>
+                                    <span>Quinn Silverman</span>
+                                </div>
+                            </div>}
+                            placement="bottom-end"
+                            className={classes.customTooltip}
+                        >
+                            <IconButton>
+                                <img src={INFO_ICON} alt="" />
+                            </IconButton>
+                        </Tooltip>
+                        <Button variant="contained" className={classes.deleteBtn} onClick={() => setOpenDeleteDialog(true)}>Delete file</Button>
+                    </Box>
+                </Box>
+                <Box display="flex" alignItems="center" justifyContent="center" py={4}>
+                    <img src={pageImg} alt='' style={{ maxWidth: 500, maxHeight: 400 }} />
+                </Box>
+            </div>
+            <Box className={classes.pagination}>
+                <Typography>
+                    Page {page} of {data[tabIdx].files[selectedIndex].pages.length}
+                </Typography>
+                <Pagination
+                    count={data[tabIdx].files[selectedIndex].pages.length}
+                    size="large"
+                    page={page}
+                    variant="outlined"
+                    shape="rounded"
+                    onChange={handlePageChange}
+                />
+            </Box>
+            <Modal
+                open={openUploadDialog}
+                dialogActions={true}
+                actionText="Upload"
+                disableGutter={true}
+                handleClose={() => setOpenUploadDialog(false)}
+                handleAction={() => { }}
+                title={`Upload PDF to Population ${populationName}`}
+            >
+                <Box display="flex" alignItems="center" justifyContent="center" p={2}>
+                    <PdfFileDrop
+                        file={files[0]}
+                        setFile={(value: any) => handleAddFile(PDF_FILE, value)}
+                        hasErrors={validationErrors.has(PDF_FILE)}
+                    />
+                </Box>
+                <CategorySelect category={category} handleCategoryChange={handleCategoryChange} />
+                {
+                    files[0] !== undefined && <Box className={classes.addAnotherFileBox}>
+                        <AddPdfFileDrop
+                            file={files[1]}
+                            setFile={(value: any) => handleAddFile(PDF_FILE, value)}
+                            hasErrors={validationErrors.has(PDF_FILE)}
+                        />
+                    </Box>
+                }
```

Comment:
This looks incorrect. The add file button should appear not only once but everytime we have the previous dropzone already with a file